flanker/addresslib/plugins/aol.py

```diff
@@ -42,7 +42,10 @@
                         ''', re.MULTILINE | re.VERBOSE)
 
 
-def validate(localpart):
+def validate(EmailAddress):
```

Comment:
According to PEP8 "CamelCase starting with a capital letter" is how classes are named in Python, variable and parameter names should consist of lowercase letters with underscore used for word separation (e.g. `email_addr`). Please make the code PEP8 compliant.